spec/requests/api/service_templates_spec.rb

```diff
@@ -19,6 +27,30 @@ def app
     Vmdb::Application
   end
 
+  describe "Service Templates query" do
+    before do
+      template.resource_actions = [ra1, ra2]
+      api_basic_authorize
+    end
+
+    it "queries all resource actions of a Service Template" do
+      run_get "#{service_templates_url(template.id)}/resource_actions", :expand => "resources"
+
+      resource_actions = template.resource_actions
+      expect_query_result(:resource_actions, resource_actions.count, resource_actions.count)
+      expect_result_resources_to_include_data("resources", "action" => resource_actions.pluck(:action))
+    end
+
+    it "queries a specific resource action of a Service Template" do
+      run_get "#{service_templates_url(template.id)}/resource_actions",
+              :expand => "resources",
+              :filter => ["action='Provision'"]
+
+      expect_query_result(:resource_actions, 1, 2)
+      expect_result_resources_to_include_data("resources", "action" => %w(Provision))
```

Comment:
wouldn't hurt, even though we're testing the action attribute itself.  Just updated and re-pushed.   Thanks.